lib/ansiblelint/runner.py

```diff
@@ -0,0 +1,94 @@
+"""Runner Module."""
+import os
+
+import ansiblelint.utils
+import ansiblelint.skip_utils
+from ansiblelint.errors import MatchError
+from ansiblelint.rules.LoadingFailureRule import LoadingFailureRule
+from typing import List, Set
```

Comment:
It's an stdlib import.

Reply:
mypy == typing, not really stdlib but close enough.